css/base.css

```diff
@@ -14,3 +14,12 @@ html {
 	overflow: hidden;
 	-webkit-user-select: none;
 }
+
```

Comment:
This code doesn't seem to be necessary, and it also affects the background color of ```<webviews>``` on pages that don't define a background color (which is a problem, since the default background color is assumed to be white). It's probably best to remove this code, and move the ```color: lightgrey``` part to ```taskOverlay.css```.